examples/cron-evals/run_evals.py

```diff
@@ -0,0 +1,45 @@
+# type: ignore
+"""
+Queries Phoenix for spans within the last minute. Computes and logs evaluations
+back to Phoenix. This script is intended to run once a minute as a cron job.
+"""
+
+from datetime import datetime, timedelta
+
+import phoenix as px
+from phoenix.evals import (
+    HallucinationEvaluator,
+    OpenAIModel,
+    QAEvaluator,
+    RelevanceEvaluator,
+    run_evals,
+)
+from phoenix.session.evaluation import get_qa_with_reference, get_retrieved_documents
+from phoenix.trace import DocumentEvaluations, SpanEvaluations
+
+phoenix_client = px.Client()
+last_eval_run_time = datetime.now() - timedelta(
+    minutes=1, seconds=10
+)  # add a few seconds to ensure all spans are captured
```

Comment:
You might want to trim the time to boundaries. Not needed now but as a follow-up

Reply:
Not sure what you had in mind here?